ui/public/config.js

```diff
@@ -1,5 +1,5 @@
 window.config = {
-  API_URL: '',
+  API_URL: 'https://api.hub.tekton.dev',
```

Comment:
yeah i think we should keep it, because whenever any new developer starts the ui locally, he should be able to see the resources, instead of a blank page

Reply:
but we should not put production api url her, could you please remove it.